### PR TITLE
Updates transcript for episode 14 (s2e7).

### PR DESCRIPTION
**Pull request by jpetto**

### CODE REVIEW

File: _posts/2018-04-03-episode-14-ctrl-alt-facts.md [hunk 1]
```diff
@@ -313,8 +313,10 @@ transcript: |
...
+
+    IRL is an original podcast from Mozilla, the not-for-profit behind the Firefox browser. I'm Veronica Belmont and i'll see online until we catch up again, IRL.
```
Comment (stephaniehobson) [inline _posts/2018-04-03-episode-14-ctrl-alt-facts.md]: `i'll`

File: _posts/2018-04-03-episode-14-ctrl-alt-facts.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ number:         14
 show_notes: |
     A recent Gallup survey found that most Americans feel that it’s harder today to be well-informed than ever before. But each of us can play a part in stopping the spread of misinformation. [Learn more](https://blog.mozilla.org/internetcitizen/2018/04/02/irl-how-to-stop-social-media-misinformation).
 transcript: |
-    **Veronica Belmont:** Hi everyone, it’s Veronica… this episode is all about misinformation and it’s our last one for this season… but also it’s the first episode we’ve ever recorded in front of a live audience.We recorded it a couple weeks ago and since then, a lot has happened on this very topic—in real life. To catch up on the latest, head to our show notes. In the meantime, enjoy this episode. Here it is.
+    **Veronica Belmont:** Hello everyone, it’s Veronica… this episode is all about misinformation and it’s our last one for this season… but also it’s the first episode we’ve ever taped in front of a live audience.We recorded it a couple weeks ago and since then, a lot has happened on this very topic—in real life. To catch up on the latest, head to our show notes. In the meantime, enjoy this episode. Here it is.
```
Comment (stephaniehobson) [inline _posts/2018-04-03-episode-14-ctrl-alt-facts.md]: `audience.We`

File: _posts/2018-04-03-episode-14-ctrl-alt-facts.md [hunk 3]
```diff
@@ -313,8 +313,10 @@ transcript: |
...
-    
+    **Veronica Belmont:** You're just making that ... The lie of that truth was better, I thought that was pretty cool. Some applause for our guests please. Thank you.This episode marks the end of our second season, but we'll be back a little later this year with plenty more storytelling for you. In fact, you can help us make the show. We want to hear from you. What kind of burning questions have you had about how the internet works, or doesn't work. The web is a deeply woven part of our lives. What do you need to understand better about how we can build a healthy internet for all? Head to the IRL website and let us know, IRLPodcast.org. And thanks to all of you, our audience, for coming IRL's first live podcast taping. I'm so glad you guys all came out. I want to leave you with one last thought. Though it’s not always easy to separate fact from fiction in social media, it’s not impossible to stop the spread of misinformation, at least from your own account. Check out the show notes for more on what you can do to promote information integrity online.
```
Comment (stephaniehobson) [inline _posts/2018-04-03-episode-14-ctrl-alt-facts.md]: `Thank you.This`

**Comment by stephaniehobson** [comment]
https://twitter.com/iamdevloper/status/397664295875805184?lang=en

I clearly don't read the transcripts when the the episodes post.

**Comment by jpetto** [comment]
> I clearly don't read the transcripts when the the episodes post.

Sigh, me neither. I skim it for obvious issues (like the timestamps that I took out of this one), but yeah, that tweet pretty much sums it up. 😞

**Comment by jpetto** [comment]
@stephaniehobson Thanks for the review - updated.

**Comment by stephaniehobson** [comment]
💯

**Pull request merged**